File: java-sample/src/me/koply/javasample/commands/SimpleCommands.java
package me.koply.javasample.commands;

import me.koply.kcommando.Command;
import me.koply.kcommando.Utils;
import me.koply.kcommando.internal.Commander;
import net.dv8tion.jda.api.events.message.MessageReceivedEvent;

@Commander(name = "Ping",
        aliases = {"ping","pingu"},
        description = "Pong!")
public class SimpleCommands extends Command {

    @Override
    public boolean handle(MessageReceivedEvent e) {
        e.getTextChannel().sendMessage(Utils.embed("Pong!")).queue();
        return true;
    }

}

File: kotlin-sample/src/me/koply/kotlinsample/commands/SimpleCommands.kt
package me.koply.kotlinsample.commands

import me.koply.kcommando.Command
import me.koply.kcommando.Utils
import me.koply.kcommando.internal.Commando
import net.dv8tion.jda.api.events.message.MessageReceivedEvent

@Commando(name = "Ping",
        aliases = ["ping", "pingu"],
        description = "Pong!")
class SimpleCommands : Command() {

    override fun handle(e: MessageReceivedEvent) : Boolean {
        e.textChannel.sendMessage(Utils.embed("Pong!")).queue()
        return true
    }

}